docs/amber_script.md

```diff
@@ -412,27 +415,53 @@ SHADER fragment kFragmentShader GLSL
   }
 END  # shader
 
-BUFFER kData TYPE vec3<int32> DATA
-# Top-left red
--1 -1  0xff0000ff
- 0 -1  0xff0000ff
--1  0  0xff0000ff
- 0  0  0xff0000ff
-# Top-right green
- 0 -1  0xff00ff00
- 1 -1  0xff00ff00
- 0  0  0xff00ff00
- 1  0  0xff00ff00
-# Bottom-left blue
--1  0  0xffff0000
- 0  0  0xffff0000
--1  1  0xffff0000
- 0  1  0xffff0000
-# Bottom-right purple
- 0  0  0xff800080
- 1  0  0xff800080
- 0  1  0xff800080
- 1  1  0xff800080
+BUFFER kPosData TYPE vec2<int32> DATA
+# Top-left
+-1 -1  
+ 0 -1  
+-1  0
+ 0  0
+# Top-right
+ 0 -1  
+ 1 -1  
+ 0  0
+ 1  0
+# Bottom-left
+-1  0
+ 0  0
+-1  1
+ 0  1
+# Bottom-right
+ 0  0
+ 1  0
+ 0  1
+ 1  1
+END
+
+BUFFER kColorData TYPE uint32 DATA
```

Comment:
The split is kinda annoying, but not too bad. Otherwise we'd need to know the locations of the different parts of the buffer which would be wonky I think.

Reply:
Better to be explicit about the different locations.  So I agree with this change.